src/apis/chat/type.d.ts

```diff
@@ -0,0 +1,32 @@
+export interface PostChatRoomReturn {
+  id: string;
+}
+
+export interface GetChatRoomReturn {
+  id: string;
+  createdAt: string;
+  updatedAt: string;
+  createdBy: number;
+  connectionId: number;
+}
+
+export interface GetChatMessageReturn {
+  totalCount: number;
+  limit: number;
+  contents: [
+    {
+      id: string;
+      createdAt: string;
+      updatedAt: string;
+      roomId: number;
+      senderId: number;
+      message: string;
+      blogPostUrl: string;
+    }
+  ];
```

Comment:
```suggestion
  contents: {
      id: string;
      createdAt: string;
      updatedAt: string;
      roomId: number;
      senderId: number;
      message: string;
      blogPostUrl: string;
    }[]
```
타입 선언시, `Array<object>` 타입은 `[ ]` 를 객체 뒤에 선언해야 한다고 합니다. 
[공식문서](https://www.typescriptlang.org/docs/handbook/2/objects.html#the-array-type)